src/core/measure.ts

```diff
@@ -0,0 +1,18 @@
+import { Metric } from "@clarity-types/data";
+import config from "@src/core/config";
+import * as metric from "@src/data/metric";
+
+// tslint:disable-next-line: ban-types
+export default function(method: Function): Function {
+    return function(): void {
+        let start = performance.now();
+        method.apply(this, arguments);
+        let duration = performance.now() - start;
+        metric.duration(Metric.Latency, duration);
+        metric.counter(Metric.InvokeCount);
+        if (duration > config.longtask) {
+            metric.counter(Metric.LongTaskCount);
```

Comment:
>counter [](start = 19, length = 7)

Minor but I prefer verb to noun names so would change these to metric.count(LongTaskCount).

Reply:
Sure, will make the change.